src/app/Components/ArtworkGrids/ArtworkGridItem.tsx

```diff
@@ -237,6 +247,11 @@ export const Artwork: React.FC<ArtworkProps> = ({
     !!priceOfferMessage.priceListedMessage &&
     !!priceOfferMessage.priceWithDiscountMessage
 
+  const displayLimitedTimeOfferSignal =
+    AREnablePartnerOfferSignals &&
+    collectorSignals?.partnerOffer?.isAvailable &&
+    !artwork.sale?.isAuction
```

Comment:
The [MP collector signals loader accounts for whether the artwork](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/collectorSignals.ts#L63-L100) is purchasable/in auction and whether the partner offer is active - so this could just be
```suggestion
  const limitedTimeOfferSignal =
    AREnablePartnerOfferSignals &&
    collectorSignals?.partnerOffer
```
And then you also have not just a truthy value but the partner offer itself in case you want the `endAt`, etc.

Reply:
@erikdstock maybe we can remove the `isAvailable` then

But for the auction artworks, maybe I'm missing something, but I don't see the check for the auction in the [partner offer condition](https://github.com/artsy/metaphysics/blob/8e21ecbaac17ea3b45885aa3f3ccc351e5a79215/src/schema/v2/artwork/collectorSignals.ts#L83) 👀